Python/cp/Longest_Palindrome_Substring.py

```diff
@@ -0,0 +1,58 @@
+def palindrome(string): 
```

Comment:
Add description of the problem at the top in commented form.

Reply:
But Sir, Vatsal Sir told me to bring it down @atarax665